index.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@import url(./Stylesheets/commons/variables.css);
+/* :root{
+    /* --primary-color:#4F46E5;
+    --dark-gray:#9CA3AF;
+    --off-white:#E5E7EB; */
+    /* --primary-color:#5265E4;
+    --dark-gray:#5265E4;
+    --off-white:#0A0A0A;
+
+
+} */ 
+
+h1{
+    color:var(--yellow-color);
+    font-size: 5rem;
+}
+
+h2,h3{
+    color:var(--yellow-color)  
+}
+
+hr{
+    margin: 2rem 0rem;
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    margin:0px;
+    background-color:var(--dark-background);
+}
+
+.container{
+    padding-left: 1rem;
+    padding-right: 1rem;
+    color:var(--primary-color);
+    font-size: 1.2rem;;
+    
+}
+
+.centered{
+    width:600px;
+    margin:auto;
+    margin-bottom: 3rem;
+    text-align: center;
+}
+
+.links{
+    text-decoration: none;
+    padding: 0.5rem 1rem;
+    border-radius: 0.5rem;
+    margin: 5rem 0rem;
+    
+}
+
+.primary-link{
+    background-color: var(--primary-color);
+    
+    color:white;
+}
+
+.links-secondary{
+    color:var(--primary-color);
+    border: 1px solid var(--primary-color);
+    border-radius: 0.5rem;
+    
+}
+
+.list-non-bullet{
+    list-style: none;
+}
+
+.list-inline{
+    display:inline;
+    padding:0rem 0.5rem;
+}
+
+.navigation{
+    background-color: var(--primary-color);
+    color:white;
+    padding:1rem;
+    border-bottom-left-radius: 1rem;
+}
+
+.list-align{
+    text-align: right;
+}
+
+.navigation .links{
+    color:white;
+}
+
+.navigation .link-active{
+    font-weight: bold;
+}
+
+.nav-brand{
+    font-weight: bold;
+    font-size:2rem
+}
+
+
+.header-img{
+    width:350px;
```

Comment:
you should consider using rems here instead of px as the width would not change with screen size and your site may look janky